src/ontology/uberon-edit.obo

```diff
@@ -224361,6 +224375,18 @@ relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 relationship: in_taxon NCBITaxon:7898 ! Actinopterygii
 property_value: dcterms-date 2022-10-31T07:56:47Z xsd:dateTime
 
+[Term]
+id: UBERON:8480041
+name: testicular sheath
+def: "Bilayer sheath that covers both the insect male mature gonad and a portion of the genital disc. It is composed of an outer pigment cell layer, an inner muscle cell layer, and their respective basal laminae." [PMID:9553045]
+synonym: "testis sheath" EXACT [] {http://purl.obolibrary.org/obo/IAO_0000232="https://www.fortunejournals.com/ijpaes/admin/php/uploads/798_pdf.pdf"}
+is_a: UBERON:0004923 ! organ component layer
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: in_taxon NCBITaxon:50557 ! Insecta
+relationship: part_of UBERON:0003135 ! male reproductive organ
+relationship: part_of UBERON:6001785 ! insect male genital disc
```

Comment:
See my comment about the definition above. I don’t think it is correct to make the sheath a part of the genital disc. It is partially derived (at least for the muscular component) from the disc and it may cover some structures that originate from the disc (such as the seminal vesicle), but it is not itself part of the disc.

You could envision making it part of `seminal vesicle`, but our current definition for that term is a little bit too vertebrate-specific. For now I would recommend making it solely part of `testis`.

Reply:
@gouttegd I made it only part of testis